<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="tasklistWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="addTaskButton">
       <property name="text">
        <string>Добавить задачу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="detailsTaskButton">
       <property name="text">
        <string>Подробности задачи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteTaskButton">
       <property name="text">
        <string>Удалить задачу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="saveTasksButton">
       <property name="text">
        <string>Сохранить данные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadTasksButton">
       <property name="text">
        <string>Загрузить данные</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
